utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -40,6 +47,20 @@ struct ThreadStats {
     classes: HashSet<&'static str>,
 }
 
+extern "C" {
+    pub fn gettid() -> u32;
+}
+
+pub fn get_tid() -> usize {
+    let res = TID.with(|t| {
+        if *t.borrow() == usize::max_value() {
```

Comment:
The initial value is 0. Am I missing something here or this condition will never be true?

Reply:
Fixed